integration/samples/secondary/feature-d/feature-d.ts

```diff
@@ -0,0 +1,5 @@
+import { FEATURE_A } from '@sample/secondary/feature-a';
```

Comment:
Can we add a spec and verify that the generate `package.json` exists and contains all the expected fields.

Such as the correct `name` and other APF fields?